demo.ipynb

```diff
@@ -0,0 +1,198 @@
+{
```

Comment:
A few comments in the last cell have been copied over but not adjusted:

* `# We display bounding boxes and the emotion class with highest predicted probability for each detected face`
* `# For each face, draw the bounding box and predicted emotion class together with the probability`
* `# Plot the emotion class and probability text`

Reply:
Fixed in 97937c7.